docs/cri/config.md

```diff
@@ -107,6 +107,16 @@ version = 2
     # set to nil or `unconfined`, and the default used when the runtime default seccomp profile is requested.
   unset_seccomp_profile = ""
 
+  # enable_unprivileged_icmp configures net.ipv4.ping_group_range="0 2147483647"
+  # for all containers which are not using host network, are not running in user namespace
+  # and if it is not overwritten by PodSandboxConfig
+  enable_unprivileged_icmp = true
```

Comment:
which version of kubernetes will this be expected to be set to true on? should probably mention that here and in the config.go..

Reply:
if you want the default to be true you need to set it in config_unix.go  you'll see the defaults there..

if the expectation is this will default on in this upcoming version of kubernetes.. I suggest making this a disable boolean with a default of false..

Reply:
if this is more alpha.. then stick with enable... and link to the feature in k8s where this will become a desired default

and stick with enable default false.. till r.next or at least a service refresh and we can change the default to true in config_unix.go when the stars align with the kubelet changes

Reply:
I understood from comments that we don't want enable these by default here because we cannot be sure that it does not break some existing use cases/configurations. That why it is now on purpose in way that those settings are not enabled by default but documentation still suggests to enabling them by showing that on examples.

Reply:
Ah, just noticed that description text says `The explanation and default value of each configuration item are as follows:` so will change those to `false`

Reply:
> if this is more alpha.. then stick with enable... and link to the feature in k8s where this will become a desired default

Not sure where you want to link? Add comment to code which links to K8s github?

Reply:
was hoping for some reference to the discussion for context to the reader ```[k8s discussion](https://github.com/kubernetes/kubernetes/issues/102612)```

Reply:
@crosbymichael note the.. default off for now.. response :-(

Reply:
Link added now.